src/backend/BSXML.pm

```diff
@@ -1650,6 +1650,7 @@ our $configuration = [
     'configuration' =>
 	[],
 	'title',        #webui only
+	'tos_url',      #webui only
```

Comment:
Sorry my editor removed the trailing spaces, this is the line the I actually added.

Reply:
You can append `?diff=split&w=1` to this PR's url to see the actual changes and discard the whitespaces